automation/src/test/scala/SignInSpec.scala

```diff
@@ -0,0 +1,31 @@
+import org.broadinstitute.dsde.firecloud.auth.{AuthToken, AuthTokens}
+import org.broadinstitute.dsde.firecloud.pages.WebBrowserSpec
+import org.broadinstitute.dsde.firecloud.{CleanUp, Config}
+import org.scalatest._
+
+class SignInSpec extends FreeSpec with WebBrowserSpec with CleanUp {
+
+  implicit val authToken: AuthToken = AuthTokens.harry
+
+  "A user" - {
+    "with a registered account" - {
+
+      "should be able to log out and log in as another user" in withWebDriver { implicit driver =>
+        val listPageAsUser1 = signIn(Config.Users.harry)
+        listPageAsUser1.signOut()
+        val listPageAsUser2 = signIn(Config.Users.ron)
+        assert(listPageAsUser2.getUser().startsWith(Config.Users.ron.email))
```

Comment:
Yes. It's all one behavior (logging out and back in as a different user). You'd still be testing one thing even with more than one assert. (I feel the "one assert" rule is more of a thought provoking exercise for people learning to how to test only one thing, not a rule to strictly adhere to.)

I'd even go as far as to say that the test that logs in as Harry then Ron then Harry should absolutely have the extra assert. Without it, the test would pass even if Harry was somehow never signed out.